data/human/human missions.txt

```diff
@@ -5416,3 +5416,99 @@ mission "Deal Change 2"
 				die
 			label skip
 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
+
+
+
+mission "Quicksilver Mixup [0]"
+	name "Package to Silver"
+	description "Deliver a package weighing <tons> to Silver by <date>. Payment is <payment>."
+	"apparent payment" 17100
+	cargo package 1
+	deadline 3 0
+	to offer
+		random < 25
+	source
+		near Algol 3
+	destination Silver
+	on offer
+		conversation
+			`You're looking at some job listings when a man with a harried face approaches you, carrying a package.`
+			`	"Excuse me, Captain," he says. "I need this package delivered in the next three days. Details are on the package. I'll pay you <payment> once it's done." Without waiting for a response, he shoves the package into your hands and runs off to attend to whatever business has gotten him in such a hurry. The package is addressed to a location on Silver.`
+			branch "can't make it in time"
+				not "hyperjumps to system: 3"
+			`	It's going to be close to his deadline, but it's possible.`
+			label "can't make it in time"
+			`	How he expects you to deliver that package in three days is beyond you, seeing as Silver is not within three jumps of <origin>, but you might as well try and get there as soon as possible.`
+			choice
```

Comment:
```suggestion
			`	How he expects you to deliver that package in three days is beyond you, seeing as Silver is not within three jumps of <origin>, but you might as well try and get there as soon as possible.`
			label choice
			choice
```

Reply:
Need to jump over the mention of an impossible deadline if you do meet it.

Reply:
Is it necessary? Even if it is possible, the point is that it went the wrong way.

Reply:
You don't know that at that point in the conversation, though. If you can make the deadline, then <origin> is within 3 jumps of Silver and it's possible. You want to skip that block of text if you get the mission in a location like that.

Reply:
Isn't that covered by 
```
		branch "can't make it in time"
				not "hyperjumps to system: 3"
			`	It's going to be close to his deadline, but it's possible.`
```
I could also add a condition for the player having a jump drive too.

Reply:
Conversations continuing going by line line if nothing interrupts them. If you're within the jumps needed, it'll say "It's going to be close, etc.", _then_ continue on to the next block of text, saying it's impossible.

But you've already made the suggestion change here of adding a `label choice`. So it's good.

Reply:
Oh, I see. : )